code/modules/integrated_electronics/subtypes/access.dm

```diff
@@ -14,6 +14,10 @@
 		"on read" = IC_PINTYPE_PULSE_OUT
 	)
 
+/obj/item/integrated_circuit/input/card_reader/old
+	name = "card reader"
+	spawn_flags = 0
```

Comment:
Not sure if this will work with old blueprints?

Reply:
I checked and it works just fine, same var is used for the old advanced medical analyser.